dataset_catalog/hytest_intake_catalog.yml

```diff
@@ -2,25 +2,25 @@ sources:
   
   conus404-hourly-onprem:
     driver: zarr
-    description: "CONUS404 Hydro Variable subset, 40 years of hourly data on Caldera on prem storage"
+    description: "CONUS404 Hydro Variable subset, 40 years of hourly data on Caldera on prem storage, updated Sep 2022"
     args:
-      urlpath: '/caldera/projects/usgs/water/wbeep/conus404_work/zarr_out/conus404_hourly.zarr/'
+      urlpath: '/caldera/hytest_scratch/scratch/conus404/conus404_hourly.zarr'
```

Comment:
Could we rename this file to `conus404_hourly_202209.zarr` to be consistent with S3? @pnorton-usgs

Reply:
And is there a reason we are working in a scratch folder?

Reply:
@amsnyder The point of the intake catalog is to hide the source datastore naming/location and provide a consistent datastore name for users. The main reason we started appending a date to the S3 data stores is because the create/modify times on S3 seem to consistently be incorrect. With the on-prem data stores we can rely on the create/modify times instead. 

As far as the use of the scratch folder is concerned, I don't know why everything is under and additional scratch directory; this just the way it was setup and I rolled with it. I'll see about moving everything up one level for the on-prem data stores.

Reply:
@amsnyder The daily and monthly zarr data stores have been transferred to S3. They can be found at:

`s3://nhgf-development/workspace/conus404_daily_20221003.zarr/`
and 
`s3://nhgf-development/workspace/conus404_monthly_20221003.zarr`

Could you transfer them to the main S3 area?  Thanks!